playbooks/roles/ocp-customization/tasks/tuned.yaml

```diff
@@ -0,0 +1,15 @@
+---
+- name: Modify kernel tunables via tuned operator
+  block:
+   - name: Include tuned vars file
+     include_vars: tuned.yaml
+
+   - name: Create tuned CRDs
+     template:
+       src: tuned-sysctl-tuning.yaml
+       dest: "{{ workdir }}/10-tuned-sysctl-tuning.yaml"
+
+   - name: Apply the tuned CRDs
+     shell: "oc apply -f {{ workdir }}/10-tuned-sysctl-tuning.yaml"
+  when: sysctl_tuned_options
```

Comment:
Move this to task/main.yaml. No need of block.

Reply:
ok